src/API/bookStoreApi.js

```diff
@@ -0,0 +1,22 @@
+// prettier-ignore
+const URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/9G5JlWMTb62xn7FV1JEu/books';
+
+export const addBookToApi = async (newBook) => {
+  await fetch(URL, {
+    method: 'POST',
+    body: JSON.stringify(newBook),
+    headers: { 'Content-type': 'application/json; charset=UTF-8' },
+  });
+};
```

Comment:
- [ ] Great job on the functionality your deleteBook and fetchBook actions are working fine but there is an issue, your addBook action is not working because your body for the post request also requires the `author` in the object. After adding a book if you refresh the page you will not see the book on the screen. Please fix this issue.
 - To fix this add author in the book object (you can add a new form field so you can have the author name from the users) 
 - Example: `
 const newBook = {
      item_id: uuidv4(),
      title,
author,
      category,
    };
`

### See the error of post request

![image](https://user-images.githubusercontent.com/83803414/162040984-f0006c38-10e1-43ca-965e-60cc73d629f2.png)

### See the API requirements for the post request.

![image](https://user-images.githubusercontent.com/83803414/162040778-866d911e-f5e5-492f-bbc2-068f73de8061.png)